.travis.yml

```diff
@@ -1,11 +1,36 @@
 language: go
 sudo: false
 go:
-  - 1.6
   - 1.7
   - 1.8
   - tip
 matrix:
   allow_failures:
     - go: tip
   fast_finish: true
+sudo: false
```

Comment:
looks like a duplicate from a few lines above. (maybe alphabetical order would help!)

Reply:
Yeah. I like the alphabetical order idea. I'm going to commit that and also add Go 1.9 while I'm at it.